app/selectors/accountTrackerController.ts

```diff
@@ -0,0 +1,21 @@
+import { createSelector } from 'reselect';
+import {
+  AccountTrackerState,
+  AccountInformation,
+} from '@metamask/assets-controllers';
+import { EngineState } from './types';
+
+const selectAccountTrackerControllerState = (state: EngineState) =>
+  state?.engine?.backgroundState?.AccountTrackerController;
+
+export const selectAccounts = createSelector(
+  selectAccountTrackerControllerState,
+  (currencyRateControllerState: AccountTrackerState) =>
+    currencyRateControllerState?.accounts,
```

Comment:
Wrong variable name referenced here:
```suggestion
  (accountTrackerControllerState: AccountTrackerState) =>
    accountTrackerControllerState?.accounts,
```